File: AcademyGestor/AcademyGestor/Models/Empresa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AcademyGestor.Modelos
{
    public class Empresa
    {
        public int id { get; set; }
        public string nombre { get; set; }
        public string razon_fiscal { get; set; }
        public string cif { get; set; }
        public string email { get; set; }
        public string telefono { get; set; }
        public string direccion { get; set; }
        public string localidad { get; set; }
        public string provincia { get; set; }
        public string cp { get; set; }

    }
}

File: AcademyGestorMobile/app/src/main/java/com/example/academygestormobile/Models/Empresa.kt
package com.example.academygestormobile.Models

data class Empresa(
    val id: Int,
    val nombre: String,
    val razon_fiscal: String,
    val cif: String,
    val email: String,
    val telefono: String,
    val direccion: String,
    val localidad: String,
    val provincia: String,
    val cp: String
)
